test/cartesian/CartesianGrid.spec.tsx

```diff
@@ -30,6 +30,179 @@ describe('<CartesianGrid />', () => {
     brushBottom: 7,
   };
 
+  const chartMargin: Margin = {
+    bottom: 11,
+    left: 12,
+    right: 13,
+    top: 14,
+  };
+
+  const expectedOffset: ChartOffset = {
+    bottom: 11,
+    brushBottom: 11,
+    height: 175,
+    left: 12,
+    right: 13,
+    top: 14,
+    width: 275,
+  };
+
+  describe('layout and size when set explicitly', () => {
+    it('should put x, y, width and height as coordinates to the background', () => {
+      const { container } = render(
+        <Surface width={500} height={500}>
+          <CartesianGrid
+            x={1}
+            y={2}
+            width={100}
+            height={200}
+            fill="green"
```

Comment:
```suggestion
```

Reply:
This seems unrelated to the test. I believe it would help to keep tests as simple as possible to make them easy to understand.

Reply:
This is required because without `fill` property, CartesianGrid does not render the background element

Reply:
But it's not required in the other two tests so I removed it from there